Sources/IO/Misc/OBJReader/index.js

```diff
@@ -195,12 +195,12 @@ function vtkOBJReader(publicAPI, model) {
     model.compression = option.compression;
 
     // Fetch metadata
-    return publicAPI.loadData();
+    return publicAPI.loadData({progressCallback: option.progressCallback});
   };
 
   // Fetch the actual data arrays
-  publicAPI.loadData = () => {
-    const promise = fetchData(model.url);
+  publicAPI.loadData = (option = {}) => {
+    const promise = fetchData(model.url, {progressCallback: option.progressCallback});
```

Comment:
Speaking of making parameter semantics clear, I'd love to be able to document them inline. I don't see any JSDoc or anything in the code, is there a preferred way to formally document function params?